### PR TITLE
Update Trademark notice

### PR DESCRIPTION
**Pull request by jlprat**
Update Trademark notice and add R icons

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,14 +1,14 @@
-Hands-on material for session on how to start working with OpenSearch and NodeJS
+Hands-on material for session on how to start working with OpenSearch® and NodeJS
 ================================================================================
```
Comment (TibsAtWork) [inline README.rst]:
```suggestion
=================================================================================
```
For reStructuredText the underlining needs to be at least as long as the title.

**Review by TibsAtWork — COMMENTED** [review]
Looks good to me, apart from the reST comment.

**Comment by jlprat** [comment]
More of a markdown person myself

**Review by TibsAtWork — APPROVED** [review]
Looks good to me. All ® seem in place at least once.

**Pull request merged**